classes/Book/label.rb

```diff
@@ -0,0 +1,15 @@
+class Label
+  attr_accessor :items, :color, :title, :id
+
+  def initialize(color, title, id = nil)
+    @id = id || Random.rand(1..1000)
+    @title = title
+    @color = color
+    @items = []
+  end
```

Comment:
I would like to kindly change the constructor to use the `params` Hash style.
- Using it would give us an standard way to initialize all of our objects in the project.
- You can get an idea looking at the `Item` class constructor.
    ```rb
      def initialize(params)
        @id = params[:id] || Random.rand(1...1_000)
        @publish_date = Date.parse(params[:publish_date])
        @archived = params[:archived] || false
      end
    ```